### ISSUE TITLE
Wrong Invoke-SFshutdown example

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#52]
```
    -------------------------- EXAMPLE 1 --------------------------
    
    PS C:\>Invoke-SFShutdown -NodeID 1,2,3,4
    
    Shuts down nodes 1 thorugh 4

```
This actually restarts the cluster.

```        
    -Option <string>
        Action to take for the node shutdown: restart or halt. Default is 
        restart.
```

**Comment by Salmanius** [comment]
Good catch Sean, Thank you.
This has been fixed on our side and will be released with 1.6.

**Comment by arjun960** [comment]
@scaleoutsean We have addressed and resolved this issue and the fix will be available in the next SDK release.
